packages/xrpl/src/models/methods/subscribe.ts

```diff
@@ -268,7 +268,17 @@ export interface TransactionStream extends BaseStream {
    */
   meta?: TransactionMetadata
   /** The definition of the transaction in JSON format. */
-  transaction: Transaction
+  transaction: Transaction & {
+    /**
+     * A unix timestamp (in seconds) marking the time of the transaction
```

Comment:
Are you sure this is a unix timestamp? I would expect "Ripple time" instead.

https://xrpl.org/basic-data-types.html#specifying-time

Reply:
Oops - just checked by looking at the time of a transaction from a few seconds ago and it is indeed "Ripple time"

Reply:
My mistake. I am not on "Ripple time"

Reply:
Most code isn't :P